api/proto/info/info.proto

```diff
@@ -783,3 +791,11 @@ message Capabilities {
   // I/O Virtualization support
   bool IOVirtualization = 3;
 }
+
+// AppMetaData sends meta data of an application instance
+// e.g kubeconfig, etc
+message ZInfoAppMetaData {
+  string uuid = 1;
+  AppMetaDataType type = 2;
+  string data = 3;
```

Comment:
Sure it should be a (UTF-8) string and not an array of arbitrary bytes?
Protobuf barfs it is it not valid UTF-8 sequences in a string.

Reply:
Let me change it to an array of bytes. Thanks!